<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlsConfig_binClass</class>
 <widget class="QMainWindow" name="ControlsConfig_binClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlsConfig_bin</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QComboBox" name="cbox_player_id">
      <item>
       <property name="text">
        <string>Player 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 8</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Player 9</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSDLInputWidget" name="conf_MoveLeft" native="true"/>
    </item>
    <item row="2" column="0">
     <widget class="QSDLInputWidget" name="conf_MoveRight" native="true"/>
    </item>
    <item row="3" column="0">
     <widget class="QSDLInputWidget" name="conf_MoveUp" native="true"/>
    </item>
    <item row="4" column="0">
     <widget class="QSDLInputWidget" name="conf_MoveDown" native="true"/>
    </item>
    <item row="5" column="0">
     <widget class="QSDLInputWidget" name="conf_Fire" native="true"/>
    </item>
    <item row="6" column="0">
     <widget class="QSDLInputWidget" name="conf_Boost" native="true"/>
    </item>
    <item row="7" column="0">
     <widget class="QSDLInputWidget" name="conf_Target" native="true"/>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="9" column="0">
     <widget class="QSDLInputWidget" name="conf_MoveXAxis" native="true"/>
    </item>
    <item row="10" column="0">
     <widget class="QSDLInputWidget" name="conf_MoveYAxis" native="true"/>
    </item>
    <item row="11" column="0">
     <widget class="QSDLInputWidget" name="conf_LookXAxis" native="true"/>
    </item>
    <item row="12" column="0">
     <widget class="QSDLInputWidget" name="conf_LookYAxis" native="true"/>
    </item>
    <item row="13" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="14" column="0">
     <widget class="QSDLInputWidget" name="conf_LookBack" native="true"/>
    </item>
    <item row="15" column="0">
     <widget class="QSDLInputWidget" name="conf_LockMovement" native="true"/>
    </item>
    <item row="16" column="0">
     <widget class="QSDLInputWidget" name="conf_LockAngle" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfiguration">
    <property name="title">
     <string>Configuration</string>
    </property>
    <addaction name="actionSave_and_Exit"/>
    <addaction name="actionReset_to_default"/>
    <addaction name="actionExit_without_Saving"/>
   </widget>
   <addaction name="menuConfiguration"/>
  </widget>
  <action name="actionSave_and_Exit">
   <property name="text">
    <string>Save and Exit</string>
   </property>
  </action>
  <action name="actionExit_without_Saving">
   <property name="text">
    <string>Exit without Saving</string>
   </property>
  </action>
  <action name="actionReset_to_default">
   <property name="text">
    <string>Reset to default</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSDLInputWidget</class>
   <extends>QWidget</extends>
   <header>qsdlinputwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="controlsconfig_bin.qrc"/>
 </resources>
 <connections/>
</ui>
